test_utils/Cargo.toml

```diff
@@ -7,5 +7,5 @@ authors = ["Nicolas Luck <nicolas@lucksus.org>"]
 holochain_dna = { path = "../dna" }
 holochain_core = { path = "../core" }
 holochain_agent = { path = "../agent" }
-wabt = "0.2.0"
+wabt = "0.4"
```

Comment:
@murrayjbrown this was already approved by the team and could be merged, but the rest of the changes in this PR need to be reviewed separately

could you please split this PR into two as the scope of entry and wabt changes seems unrelated to each other?

Reply:
@thedavidmeister  Yeah, sorry about that. Will do.